sparkles/core.py

```diff
@@ -381,10 +371,57 @@ def get_acas_from_pickle(load_name, loud=False):
             break
     else:
         raise FileNotFoundError(f'no matching pickle file {filenames}')
+    return acas_dict, pth.name
+
+
+def get_acas_dict_from_occweb(path):
+    """ Get pickle file from OCCweb"""
+    from kadi.occweb import get_occweb_dir, get_occweb_page
+
+    if not path.endswith('.pkl.gz'):
+        occweb_files = get_occweb_dir(path)
+        pkl_files = [name for name in occweb_files['Name'] if name.endswith('.pkl.gz')]
+        if len(pkl_files) > 1:
+            raise ValueError(f'Found multiple pickle files in {path}')
```

Comment:
In my short experience., multiple pickles in the same directory is very common.

If there is more than one prelim, the second one will often be in the same directory with a different name. If we are going to make this change (which I like), we might as well prompt the user with options.

Reply:
Do they usually give a full path once there are multiple pickles? I guess the prompt is another option which could work OK since this is typically an interactive situation.

Reply:
A typical answer is like this one from Julia to Jean:

> I’ve placed a Pickle file with the creep-away attitude included in the same place as before (named “FEB2122_intermediate”) if you want to take a look.

In a previous message in the thread ona can get the location:

> \\noodle\GRETA\mission\Backstop\ScheduleWorkspaces\FEB2122\aca

so basically one would try that location, get multiple options, and one will have the `FEB2122_intermediate` in the name.

A similar exchange has happened before. This was just the latest.

Reply:
but true, in this case I could just concatenate the two myself.

Reply:
One problem is testing, I couldn't find any example in there now with multiple pickles. Maybe if the exception message just lists the available options and then it is pretty easy to copy/paste.

Reply:
I was thinking something like:
```
option_str = [f'\n  {i}. {o}' for i, o in enumerate(pkl_files)]
i = input(f'Choose one file:{"".join(option_str)}\n')
pkl_files = [pkl_files[i]]
```
but just listing is ok.

Reply:
Another option would just be to ask FOT MP to use a different directory for each run.

Reply:
See ece5558 and the new functional test documented in the description.

Reply:
We don't want to impose process rule number 852 on FOT MP to save a few lines of code!